From c9d26e763bf444c276e65ae84bd3cf7fff23aba7 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Tue, 17 Oct 2017 16:27:26 +0800
Subject: [PATCH] PM / devfreq: rockchip_dmc: Fix compiler warning

drivers/devfreq/rockchip_dmc.c: In function 'rk3368_dmc_init':
drivers/devfreq/rockchip_dmc.c:649:7: warning: 'ret' may be used uninitialized in this function [-Wuninitialized]
drivers/devfreq/rockchip_dmc.c:639:6: note: 'ret' was declared here

Change-Id: I843b7a4a0804bcf02152ba3218dcefa26d047c95
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 drivers/devfreq/rockchip_dmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/devfreq/rockchip_dmc.c b/drivers/devfreq/rockchip_dmc.c
index 12690641b4e3..98ed4f181726 100644
--- a/drivers/devfreq/rockchip_dmc.c
+++ b/drivers/devfreq/rockchip_dmc.c
@@ -636,7 +636,7 @@ static struct rk3368_dram_timing *of_get_rk3368_timings(struct device *dev,
 {
 	struct rk3368_dram_timing *timing = NULL;
 	struct device_node *np_tim;
-	int ret;
+	int ret = 0;
 
 	np_tim = of_parse_phandle(np, "ddr_timing", 0);
 	if (np_tim) {
-- 
2.35.3

